src/main/java/uk/gov/hmcts/sptribs/document/content/DraftEditTemplateContentCIC.java

```diff
@@ -0,0 +1,137 @@
+//package uk.gov.hmcts.sptribs.document.content;
+//
```

Comment:
Why is everything commented out?

Reply:
this is very old code coming from master.I updated from master and now my branch has very old code .Dnt know why